simplq/amplify/backend/auth/userPoolGroups/user-pool-group-precedence.json

```diff
@@ -1,6 +1,6 @@
 [
-    {
-        "groupName": "dev",
-        "precedence": 1
-    }
-]
\ No newline at end of file
+  {
```

Comment:
Amplify changes, are they just formatting changes, or are you also trying something out?